monitor_ingest/config.schema.yaml

```diff
@@ -0,0 +1,137 @@
+---
+action_type:
+  description: "Action type to be used for the entity"
+  type: "string"
+  enum:
+    - SetupEntityAction
+    - SetupAddConstants
+    - SetupAddDimesions
+    - SetupAddFunctions
+  required: false
+entity_name:
+  description: "Entity Name is must in case of action_type : LoadCsv"
+  type: "string"
+data_file_path:
```

Comment:
Where does the file identified here come from?

Reply:
@blag It is the same file which is residing on disk. This variable here is pointing to the path of the file.